### PR TITLE
[GH page] Improve texts in index.html

### PR DESCRIPTION
**Pull request by freq-mod**
less awkward, I guess

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -85,31 +85,31 @@ <h5 class="card-title">Multi-platform</h5>
...
+                  <p class="card-text">FM sliders and instrument sequential macros can be edited using mouse.</p>
                 </div>
               </div>
             </div>
             <div class="col">
               <div class="card border-0 h-100 shadow">
                 <div class="card-body">
                   <h5 class="card-title">MIDI input</h5>
-                  <p class="card-text">Entry notes from your MIDI device.</p>
+                  <p class="card-text">Entry notes with your MIDI device is possible.</p>
```
Comment (OPNA2608) [inline index.html:104]: Maybe
```suggestion
                  <p class="card-text">Note input via a MIDI device is supported.</p>
```

**Review by OPNA2608 — CHANGES_REQUESTED** [review]

File: index.html (hunk 1 above)
Comment (freq-mod) [reply index.html]: thanks

**Comment by rerrahkr** [comment]
Thanks for your contribution!
I just made a simple web page, so feel free to edit it more if you can create a clearer layout or content.

**Comment by freq-mod** [comment]
Will certainly think about it, there is some room for improvement, the more I look at it...

**Pull request merged**